analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CollectionEmptinessChecking.Fixed.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Tests.Diagnostics
+{
+    public class CollectionEmptinessChecking
+    {
+        private static bool HasContent1(IEnumerable<string> l)
+        {
+            return l.Any; // Fixed
+        }
+        private static bool HasContent1b(IEnumerable<string> l)
+        {
+            return l.Any; // Fixed
+        }
+        private static bool HasContent2(List<string> l)
+        {
+            return Enumerable.Any; // Fixed
```

Comment:
This is not a correct fix for the original line
```suggestion
            return l.Any(); // Fixed
```

Reply:
This still doesn't produce `l.Any`. I would not be happy with `Enumerable.Any(l)` syntax in my code :/

Reply:
Sorry for the misunderstanding. If I typed Enumerable.Count(l), i would argue that was intended. But I'm happy to convert that expression too.

Reply:
Let's try to convert it. 
The current implementation is rather short and clean. If it explodes into something too complicated, we can stop and keep it as it is.

Reply:
See implementation: it works, but is less elegant.